cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -44,6 +44,14 @@ const (
 	defaultContainerdConfigFilePath = "/etc/containerd/config.toml"
 	defaultCrioConfigFilePath       = "/etc/crio/crio.conf"
 	defaultDockerConfigFilePath     = "/etc/docker/daemon.json"
+
+	defaultConfigSource = configSourceCommand
```

Comment:
This changes the default behaviour for this command. Is this something we want to do with this change? If so, it should be properly called out in the commit message / release notes.

Would you be open to keeping the default `file` for now and then switch to command at a later stage?